CloudyCider/SceneDelegate.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SceneDelegate.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/1/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+  var window: UIWindow?
+
+  func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+    // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+    // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+    // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+
+    #if targetEnvironment(UIKitForMac)
+    let windowScene = UIWindowScene(session: session, connectionOptions: connectionOptions)
+    let window = UIWindow(windowScene: windowScene)
+    #else
+    let window = UIWindow(frame: UIScreen.main.bounds)
+    #endif
+    let ec2Store = EC2Store()
+    let controller = UIHostingController(rootView: HomeView().environmentObject(ec2Store))
```

Comment:
is there a specific reason to create a new `EC2Store` here and pass as en environment object in addition to the `EC2Store` created within `EC2Screen`?